src/ZoneServer/Network/Send.Normal.cs

```diff
@@ -245,6 +246,50 @@ public static void SkillProjectile(ICombatEntity entity, string packetString1, T
 				entity.Map.Broadcast(packet, entity);
 			}
 
+			/// <summary>
+			/// Plays an animation of a character throwing one of their items
+			/// </summary>
+			/// <param name="character"></param>
+			/// <param name="str"></param>
+			/// <param name="str2"></param>
+			/// <param name="position"></param>
+			/// <param name="animationId"></param>
+			/// <param name="scale"></param>
+			/// <param name="tossScale"></param>
+			/// <param name="hangScale"></param>
+			/// <param name="speed"></param>
+			/// <param name="f5"></param>
+			/// <param name="f6"></param>
+			/// <param name="f7"></param>
+			/// <param name="itemScale"></param>
+			/// <param name="itemAppearOnFloorDuration"></param>
```

Comment:
I changed it to itemStayOnGroundSeconds, which I think is more descriptive and also slightly shorter.  I actually don't really like the idea of using a timespan here, because we just convert it right back to a float, it seems like unnecessary overhead.

Reply:
That's better. Though I do want to say, while it undoubtedly takes cycles to turn the TimeSpan into a float, it shouldn't be understated how much it improves usability. You don't use it because it's faster, but because there are no questions about units and no magic numbers, it's clear and to the point. And I would argue that's generally much more important than saving fractions of a nanosecond on the cast to and from a TimeSpan^^